trp_external_user/__init__.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
```

Comment:
Please remove Copyright lines from init files altoghether, according to present OCA guidelines.